geocoding/verify.py

```diff
@@ -0,0 +1,20 @@
+'''
```

Comment:
Excellent idea! Perhaps this should be called `tests`?

Reply:
Renamed to `tests`